Resources/Prototypes/Entities/Structures/Walls/girders.yml

```diff
@@ -6,7 +6,7 @@
   components:
   - type: Transform
     anchored: true
-    noRot: true
+    noRot: false
```

Comment:
BaseStructureDynamic is parented by many many items, so this would likely cause a lot of breakage. If this is important for your changes, add noRot to the new prototypes that require them.